include/cutlass/fp8_to_fp16.h

```diff
@@ -42,6 +42,8 @@
 
 using uchar8 = cute::intel::uchar8;
 using ushort8 = cute::intel::ushort8;
+using uchar16 = cute::intel::uchar16;
+using ushort16 = cute::intel::ushort16;
```

Comment:
Can you please rebase to the main branch?
This file will be refactored by two PRs, anyway, so perhaps it's fine to rebase after they land.